lib/IRGen/GenProto.cpp

```diff
@@ -1775,7 +1790,10 @@ class AccessorConformanceInfo : public ConformanceInfo {
       if (IGM.Context.LangOpts.hasFeature(Feature::Embedded)) {
         // In Embedded Swift associated-conformance entries simply point to the witness table
         // of the associated conformance.
-        llvm::Constant *witnessEntry = IGM.getAddrOfWitnessTable(associatedConformance.getConcrete());
+        ProtocolConformanceRef assocConf =
```

Comment:
Is it possible for assocConf to differ from associatedConformance above?

Reply:
Yes, because `ConformanceInContext` is not the same as `SILWT->getConformance()`.
(I tried that first and it didn't work).